Berksfile

```diff
@@ -2,3 +2,6 @@
 source 'https://supermarket.chef.io'
 
 metadata
+
+cookbook 'apt'
+cookbook 'yum'
```

Comment:
It looks like 'git' is just going to be used during testing/validation.  In that case, the better pattern is to use a fixture cookbook to install git rather than a 'prep' recipe.  See this for example of install 'java' for testing the 'jenkins' cookbook: 
https://github.com/chef-cookbooks/jenkins/blob/master/Berksfile#L9-L21

https://github.com/chef-cookbooks/jenkins/blob/master/test/fixtures/cookbooks/jenkins_server_wrapper/recipes/default.rb#L3